### ISSUE TITLE
Auto creation of the “temp" dir.

### ISSUE THREAD

**Issue by ahowardsky** [balderdashy/skipper-s3#3]
I got the following error when testing on windows.

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: ENOENT, open 'e:\windows\Temp\mpu-\test.jpg-daa2-1407058849110-1'

It seems to be fine after I manually created the temp dir "e:\windows\Temp\mpu-\".
Would it be better if skipper-s3 could automatically create the dir if it does not pre-exist?

Thanks,
Howard

**Comment by sailsbot** [comment]
Thanks for posting, @ahowardsky.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by jkeczan** [comment]
I believe that this issue still persists, see this new issue

https://github.com/balderdashy/skipper-s3/issues/35
